### DISCUSSION TITLE
I have legitimate legal access to openfile PDF but I am anti adobe is it possible to open them in sumtrapdf?

### DISCUSSION THREAD

**Discussion by jamiecropley** [sumatrapdfreader/sumatrapdf#4337] [Q&A] [answered]
I have legitimate legal access to openfile PDF's but I am anti adobe is it possible to open them in sumtrapdf?

https://plugin.fileopen.com/

They seem to require adobe acrobat, but I do not like adobe products for various reasons.

I really need to open these PDF's for academic research.

Do not confuse my question here either, I am not asking to remove any DRM, if anything just trying to open it with the DRM in non adobe software.

**Comment by GitHubRulesOK** [comment, marked as answer]
The PDF DRM is Adobe proprietary coding system, so can only be opened in Adobe Licensed software.

>System Requirements
64-bit Windows 10 or later, Server 2016 or later
Adobe Reader/Adobe Acrobat 9.0 or later

... the publisher requires you to install FileOpen plugin to access the file. Note that installation of the FileOpen Client software is not sufficient to enable opening of documents, the publisher's permission is also required.

Thus the FileOpen non PDFs are not Open Licensed they do not follow Accessibility by any other readers, 
best alternative may be an executable by Adobe Allies say Foxit?

**Reply by jamiecropley** [reply]
I will give it a try, thanks!

**Reply by GitHubRulesOK** [reply]
Foxit download for FileOpen is here https://www.foxit.com/pdf-editor/addons.html

File Open blog says
>FileOpen offers support for a number of other viewers (Foxit, Nuance, Nitro, Bluebeam, Tracker, etc.)
 clearly SumatraPDF is more secured than those, so there is no JS or any other way such a plug-in will run without FileOpen write one
